python/ray/tests/accelerators/test_intel_gpu.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sys
+import pytest
+from unittest.mock import patch
+
+import ray
+from ray._private.accelerators import IntelGPUAcceleratorManager as Accelerator
+from ray.util.accelerators import INTEL_MAX_1550, INTEL_MAX_1100
+
+
+def test_visible_intel_gpu_ids(shutdown_only):
+    with patch.object(Accelerator, "get_current_node_num_accelerators", return_value=4):
+        os.environ["ONEAPI_DEVICE_SELECTOR"] = "level_zero:0,1,2"
+        ray.init()
+        manager = ray._private.accelerators.get_accelerator_manager_for_resource("GPU")
+        assert manager.get_current_node_num_accelerators() == 4
+        assert manager.__name__ == "IntelGPUAcceleratorManager"
+        assert ray.available_resources()["GPU"] == 3
+
+
+def test_visible_intel_gpu_type(shutdown_only):
+    with patch.object(
+        Accelerator, "get_current_node_num_accelerators", return_value=4
+    ), patch.object(
+        Accelerator, "get_current_node_accelerator_type", return_value=INTEL_MAX_1550
+    ):
+        os.environ["ONEAPI_DEVICE_SELECTOR"] = "level_zero:0,1,2"
+        ray.init()
+        manager = ray._private.accelerators.get_accelerator_manager_for_resource("GPU")
+        assert manager.get_current_node_accelerator_type() == INTEL_MAX_1550
+
+
+def test_get_current_node_num_accelerators():
+    sys.modules["dpctl"] = __import__("mock_dpctl_1")
```

Comment:
should we reset it to old value after test is done so it doesn't affect other tests?

Reply:
fixed.